src/org/jgroups/protocols/FILE_PING.java

```diff
@@ -141,8 +141,7 @@ protected boolean addDiscoveryResponseToCaches(Address mbr, String logical_name,
     protected static String addressToFilename(Address mbr) {
         String logical_name=UUID.get(mbr);
         return (addressAsString(mbr) + (logical_name != null? "." + logical_name + SUFFIX : SUFFIX))
-          .replace(File.separatorChar, '-')
-          .replace('/', '-');
+          .replaceAll("[\0<>:\"/\\|?*]", "-");
```

Comment:
Good idea, done (master and 3.6 branch).
Wow, that was quick, someone *is* paying attention here! :-)